backend/src/main/java/in/kahl/promptwhispers/controller/GameController.java

```diff
@@ -7,7 +7,7 @@
 import org.springframework.web.bind.annotation.*;
 
 @RestController
-@RequestMapping("/api/game")
+@RequestMapping("/api/games")
```

Comment:
Kleiner Tipp: Der PR heißt simple google login. Auch wenn das hier eine kleinigkeit ist und man denkt, man könnte es einfach mit in den PR übernehmen, sieht die Realität leider anders aus. Da dürftest du so eine simple Änderung nicht einfach in einen PR, der damit nicht zu tun hat, verpacken. Ist jetzt nicht schlimm, nur für später und vielleicht für künftige PRs:)

Reply:
Dank dir für die Empfehlung. Ich hatte mir sowas schon gedacht. Daher gut, dies noch mal explizit zu hören.